gc/base/standard/ParallelGlobalGC.cpp

```diff
@@ -1908,3 +1912,17 @@ MM_ParallelGlobalGC::healHeap(MM_EnvironmentBase *env)
 	 */
 }
 #endif /* defined(OMR_ENV_DATA64) && defined(OMR_GC_FULL_POINTERS) */
+
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+bool
+MM_ParallelGlobalGC::reinitializeForRestore(MM_EnvironmentBase *env)
+{
+	bool result = true;
+	/* sweepHeapSectioning should eventually be reinitialized through the collector's _sweepScheme. */
```

Comment:
not sure what you mean by this comment - what else will happen eventually?

Reply:
Yes its not clear or correct, I meant to say  "Consider reinitializing through the collector's _sweepScheme"

Reply:
Its a bit of an overkill to have indirection, it may make sense later on, hence the comment

Reply:
Updated the comment.